cfg/atari-cart.cfg

```diff
@@ -10,23 +10,25 @@ SYMBOLS {
     __CARTFLAGS__:       type = weak,   value = $01;   # see documentation for other possible values
 }
 MEMORY {
-    ZP:     file = "", define = yes, start = $0082,                size = $007E;
-    MAIN:   file = "", define = yes, start = %S,                   size = __CARTSIZE__;
-    ROM:    file = %O, define = yes, start = $C000 - __CARTSIZE__, size = __CARTSIZE__ - 6, fill = yes, fillval = $FF;
-    CARTID: file = %O,               start = $BFFA,                size = $0006;
+    ZP:      file = "", define = yes, start = $0082,                size = $007E;
+    MAIN:    file = "", define = yes, start = %S,                   size = __CARTSIZE__;
+    ROM:     file = %O, define = yes, start = $C000 - __CARTSIZE__, size = __CARTSIZE__ - 6, fill = yes, fillval = $FF;
+    CARTID:  file = %O,               start = $BFFA,                size = $0006;
+    DISCARD: file = "",               start = $0100,                size = $FF00;
 }
 SEGMENTS {
-    ZEROPAGE: load = ZP,              type = zp,                optional = yes;
-    EXTZP:    load = ZP,              type = zp,                optional = yes;
-    STARTUP:  load = ROM,             type = ro,  define = yes, optional = yes;
-    LOWCODE:  load = ROM,             type = ro,  define = yes, optional = yes;
-    ONCE:     load = ROM,             type = ro,                optional = yes;
-    CODE:     load = ROM,             type = ro,  define = yes;
-    RODATA:   load = ROM,             type = ro,                optional = yes;
-    DATA:     load = ROM, run = MAIN, type = rw,  define = yes, optional = yes;
-    INIT:     load = MAIN,            type = bss,               optional = yes;
-    BSS:      load = MAIN,            type = bss, define = yes, optional = yes;
-    CARTHDR:  load = CARTID,          type = ro;
+    ZEROPAGE: load = ZP,                  type = zp,                optional = yes;
+    EXTZP:    load = ZP,                  type = zp,                optional = yes;
+    STARTUP:  load = ROM,                 type = ro,  define = yes, optional = yes;
+    LOWBSS:   load = DISCARD, run = MAIN, type = rw,                optional = yes;  # not zero-initialized
```

Comment:
The `DISCARD:` memory area isn't needed.
`load = MAIN, type =` has the same effect that `load = DISCARD, run = MAIN, type =` has.

That `load = MAIN,` means `load = MAIN, run = MAIN,`.
The "load" parts have no effect (because of `file = ""`); the "run" parts are the only things that matter there (they are the same).